ci/utils.sc

```diff
@@ -136,3 +136,20 @@ def escapeCmdArg(cmd: String): String = {
   val subbed = cmd.replace("'", "\\'").replace("\n", "\\n")
   s"""$$'${subbed}'"""
 }
+
+case class SemVer(major: Int, minor: Int, build: Int, commit: String) {
```

Comment:
Do you think we should rename this version, since it's no longer really a semantic version? Perhaps MarathonVersion?

Reply:
I'll rename it with the forward port of your changes.